src/components/ProductList.tsx

```diff
@@ -70,7 +69,14 @@ const ProductList = () => {
       <div className="mb-8">
         <ProductSearch onSearch={handleSearch} />
       </div>
+
       
+      <div className="mb-6 text-right text-slate-700 mt-6">
+        <p>
+          {t("عدد المنتجات التي تظهر هو")}: <span className="font-bold">{filteredProducts.length}</span>
```

Comment:
Nice idea! While it's not a priority right now, feel free to open an issue for it so we can revisit this later.